src/enums.js

```diff
@@ -52,4 +52,10 @@ const MenuItem = {
   STATS: 'Stats',
 };
 
-export { RenderPosition, EventType, Filter, Sorting, Keydown, UserAction, UpdateType, MenuItem };
+const StorageTypes = {
```

Comment:
```suggestion
const StorageType = {
```
Название перечисления должно быть в единственном числе

Reply:
исправила